frontend/rolecall/src/app/cast/cast-drag-and-drop.component.spec.ts

```diff
@@ -1,15 +1,14 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { CastDragAndDrop } from './cast-drag-and-drop.component';
-
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {CastDragAndDrop} from './cast-drag-and-drop.component';
 
 describe('CastDragAndDropComponent', () => {
   let component: CastDragAndDrop;
   let fixture: ComponentFixture<CastDragAndDrop>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [CastDragAndDrop]
-    })
+        declarations: [CastDragAndDrop]
```

Comment:
I am trying to understand the logic for this indentation change. Is 'declaration' indented once for the '(' and once for the '{'?

Reply:
That's an interesting case, I had to double-check it myself. Looks like it's not from the parenthesis, but because it is followed by the chained `.compileComponents()` call, which caused a 4-space indent and this line indented to line it up. Without a chained call it ends up being just the regular 2-space indent.